src/event-stream/event-stream.service.ts

```diff
@@ -193,13 +199,27 @@ export class EventStreamService {
       batchSize: 50,
       batchTimeoutMS: 500,
       type: 'websocket',
-      websocket: { topic },
+      websocket: { topic: name },
       blockedReryDelaySec: 30, // intentional due to spelling error in ethconnect
       inputs: true,
       timestamps: true,
     };
 
     const existingStreams = await this.getStreams(ctx);
+
+    // Check to see if there is a deprecated stream that we should remove
+    this.logger.debug(`Checking for deprecated event steam with topic '${topic}'`);
+    const deprecatedStream = existingStreams.find(s => s.name === topic);
+    if (deprecatedStream) {
+      this.logger.debug(`Purging deprecated eventstream '${deprecatedStream.id}'`);
```

Comment:
This feels important, probably `log` or higher...

Reply:
I know you know, but it needs to be really clear in the release notes that this release will delete and recreate/replay all of your subscriptions.

Reply:
It is in the release notes already